web/massastation/src/layouts/LayoutStation/LayoutStation.tsx

```diff
@@ -0,0 +1,98 @@
+import { ReactNode, useState } from 'react';
+import { useLocalStorage } from '../../custom/useLocalStorage';
+
+import { useResource } from '../../custom/api';
+import { AccountObject } from '../../models/AccountModel';
+import { useAccountStore } from '../../store/store';
+import { URL } from '../../const/url/url';
+
+import {
+  ThemeMode,
+  StationLogo,
+  Dropdown,
+  Identicon,
+} from '@massalabs/react-ui-kit';
+
+export interface LayoutStationProps {
+  children?: ReactNode;
+  navigator?: Navigator;
+  logo?: ReactNode;
+  onSetTheme?: () => void;
+  activePage: string;
+  customClass?: string;
+  overwrittenTheme?: string;
```

Comment:
...do we need that!? 🤔

Reply:
I need activePage, but other you're right